### PR TITLE
fix: 🐛 fix instruction from middleware with SQ v19+

### PR DESCRIPTION
**Pull request by polymath-eric**
### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/api/entities/Instruction/index.ts [hunk 1]
```diff
@@ -355,7 +355,7 @@ export class Instruction extends Entity<UniqueIdentifiers, string> {
         },
       },
     } = await context.queryMiddleware<Ensured<Query, 'instructions'>>(
-      instructionsQuery(false, {
+      instructionsQuery(context.isSqIdPadded, {
         id: id.toString(),
```
Comment (F-OBrien) [inline src/api/entities/Instruction/index.ts:359]: In SQ 19 instruction IDs also are padded so this needs conditional padding.

**Comment by prashantasdeveloper** [comment]
Triggered from https://github.com/PolymeshAssociation/polymesh-sdk/pull/1410#issuecomment-2531595177 by [@&ZeroWidthSpace;polymath-eric](https://github.com/polymath-eric).

Trying to  fast forward `alpha` (8260aacd31d78e376aea5dee6adbbc46bb039762) to `fix-sq-instruction` (0169504cb2a7b80a5bf9b0ea434187e72dbf0fbb).

Target branch (`alpha`):

```shell
commit 8260aacd31d78e376aea5dee6adbbc46bb039762 (HEAD -> alpha, tag: v27.0.3-alpha.1, origin/alpha)
Author: Eric Richardson <eric@polymesh.network>
Date:   Mon Dec 9 09:32:26 2024 -0500

    docs: ✏️ update deprecated note to reference correct field
    
    Update deprecated notices to include "Created" where needed, e.g.
    `BlockIdDesc` -> `CreatedBlockIdDesc`
```

Pull request (`fix-sq-instruction`):

```shell
commit 0169504cb2a7b80a5bf9b0ea434187e72dbf0fbb (pull_request/fix-sq-instruction)
Author: Eric Richardson <eric@polymesh.network>
Date:   Mon Dec 9 19:42:48 2024 -0500

    fix: 🐛 fix instruction from middleware with SQ v19+
```
Fast forwarding `alpha` (8260aacd31d78e376aea5dee6adbbc46bb039762) to `fix-sq-instruction` (0169504cb2a7b80a5bf9b0ea434187e72dbf0fbb).
```shell
$ git push origin 0169504cb2a7b80a5bf9b0ea434187e72dbf0fbb:alpha
remote: error: GH006: Protected branch update failed for refs/heads/alpha.        
remote: 
remote: - All comments must be resolved.        
To https://github.com/PolymeshAssociation/polymesh-sdk.git
 ! [remote rejected]   0169504cb2a7b80a5bf9b0ea434187e72dbf0fbb -> alpha (protected branch hook declined)
error: failed to push some refs to 'https://github.com/PolymeshAssociation/polymesh-sdk.git'

File: src/api/entities/Instruction/index.ts (hunk 1 above)
Comment (polymath-eric) [reply src/api/entities/Instruction/index.ts:359]: Thanks. I added handling code in the query now

**Review by F-OBrien — APPROVED** [review]

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**